src/proto/mapping_options.proto

```diff
@@ -42,25 +42,31 @@ message MappingOptionsProto {
   required bool fix_parameters_before_scheduling = 3;
   // Tile sizes.  If empty or not provided, do not tile.
   optional TilingProto tiling = 4;
-  // Block dimensions (min 1, max 3 values).
-  required CudaDimProto block = 5;
-  // Grid dimensions (min 1, max 3 values).
-  required CudaDimProto grid = 6;
   // Unroll innermost loops to until at most "unroll" copies.  If not provided,
   // do not unroll.
   optional uint64 unroll = 7;
   // Apply tiling to imperfectly nested loops if possible.  Same tilesizes are
   // used.
   required bool tile_imperfectly_nested = 8;
+  // reserved 5, 6, 9 to 13; can only activate with proto3
+  // Reserved: 5, 6, 9-13 -> factored out into CudaMappingOptionsProto
+  required bool match_library_calls = 14;
+}
+
+message CudaMappingOptionsProto {
```

Comment:
@nicolasvasilache , looks like I misread this part before. The split actually looks good here. I.e. fields are not being flattened out. Somehow initially I thought that this file is not changed.